packages/ubuntu_wsl_setup/lib/wizard.dart

```diff
@@ -29,17 +29,8 @@ class UbuntuWslSetupWizard extends StatelessWidget {
         Routes.selectLanguage: const WizardRoute(
           builder: SelectLanguagePage.create,
         ),
-        Routes.profileSetup: WizardRoute(
+        Routes.profileSetup: const WizardRoute(
           builder: ProfileSetupPage.create,
-          onNext: (settings) {
-            if ((settings.arguments as bool?) == true) {
-              return Routes.advancedSetup;
-            }
```

Comment:
would it be enough to remove this part and let it unconditionally jump to `Routes.applyingChanges`? could we that way keep `Routes.advancedSetup` in the list of known routes?

Reply:
Yeah, that's doable.

Reply:
Done